gradle/libs.versions.toml

```diff
@@ -0,0 +1,93 @@
+[versions]
```

Comment:
@MateuszMroz czy to sposób na definiowanie zależności rekomendowanym sposobem pod dependabota? Wydaje mi się, że gdzieś o tym czytałem, ale nie miałem jeszcze okazji sprawdzić w akcji.

Reply:
Tak, żeby sprawdzić czy pojawiła się nowa wersja dependencji trzeba odpalić skrypt i właśnie to bede musiał dorobić :D 
Ogolnie spoko, dobrze sprawdza sie w przypadku modularyzacji, dodatkowo schludnie wygląda i wszystko w jednym miejscu. 

Dla jednych skrypt sprawdzajacy moze byc uciązliwy, ale w wiekszych projektach to zbawienie :)

Tutaj znajdziesz wiecej informacji: https://sourcediving.com/manage-your-gradle-dependencies-with-version-catalog-not-only-in-android-114117647cdb